### PR TITLE
add some other awesome links

### PR DESCRIPTION
**Pull request by 06kellyjac**
Added:

- flake-utils
- devshell
- PR tracker
- rust-overlay

### CODE REVIEW

**Comment by houstdav000** [comment]
They look like good contributions. Could you rearrange where you put them to keep the lists alphabetical though?

**Comment by 06kellyjac** [comment]
Sorry I didnt notice they were alphabetical. I'll resolve that

File: README.md [hunk 1]
```diff
@@ -167,6 +170,7 @@
 * [nixpkgs-mozilla](https://github.com/mozilla/nixpkgs-mozilla) - Mozilla's overlay with bleeding-edge Rust and Firefox.
 * [nixpkgs-wayland](https://github.com/colemickens/nixpkgs-wayland) - Bleeding-edge Wayland packages.
 * [NUR](https://github.com/nix-community/NUR/) - Nix User Repositories. The mother of all overlays, allowing access to user repositories and installing packages via attributes.
+* [rust-overlay](https://github.com/oxalica/rust-overlay) - A compatible replacement for Mozilla's overlay that's pure and reproducible.
```
Comment (Mic92) [inline README.md:173]: Should this be in the rust section? We could do the same with the nixpkgs-mozilla one.

Comment (06kellyjac) [reply README.md:173]: Yeah I can move them both

**Pull request merged**